docs/MCP_TOOLS.md

```
@@ -13,12 +13,12 @@ Every tool the LLM can call via MCP, grouped by service. The **REST** column sho
 - [Google Drive](#google-drive) (15)
 - [Gmail](#gmail) (14)
 - [Google Slides](#google-slides) (6)
-- [ClickUp](#clickup) (43)
-- [Slack (bot)](#slack-bot-) (7)
-- [Slack (user)](#slack-user-) (7)
+- [ClickUp](#clickup) (45)
+- [Slack (bot)](#slack-bot-) (8)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)